typescript/sdk/src/index.ts

```diff
@@ -172,3 +172,5 @@ export {
 } from './utils/objects';
 export { delay } from './utils/time';
 export { chainMetadataToWagmiChain } from './utils/wagmi';
+export { HookConfig } from './hook/types';
+export { HyperlaneHookDeployer } from './hook/HyperlaneHookDeployer';
```

Comment:
The exports are ordered alphabetically, which VSCode/prettier should do for you on save. Maybe you're missing a vscode setting? Here's what my typescript settings look like:
```json
  "[typescript]":{
     "editor.defaultFormatter":"esbenp.prettier-vscode",
     "editor.formatOnSave":true,
     "editor.codeActionsOnSave": {
         "source.organizeImports": true
      }
  },
  ```